src/ripple/app/main/Main.cpp

```diff
@@ -49,6 +49,7 @@
 #include <cstdlib>
 #include <iostream>
 #include <utility>
+#include <stdexcept>
```

Comment:
My guidance is to #include anything you use or name directly.  I found a use of `std::domain_error` in Main.cpp, so I recommend keeping `#include<stdexcept>`.